shared/test/otSdkDriver_test.js

```diff
@@ -686,52 +686,6 @@ describe("loop.OTSdkDriver", function() {
         driver.CONNECTION_START_TIME_ALREADY_NOTED);
     });
 
-    it("should record telemetry with SHORTER_THAN_10S for calls less than 10s", function() {
```

Comment:
So we should be able to remove all of the `_noteConnectionLengthIfNeeded` tests - though I'm not seeing the changes for otSdkDriver.js in this PR.

There should be quite a bit of other code associated with the connection lengths as well - e.g. search for `TWO_WAY_MEDIA_CONN_LENGTH`

Additionally, we should be able to remove the `local_check_media_start_time_uninitialized`, `local_check_media_start_time_initialized` and `local_check_connection_length_noted` from the functional tests - which means we can also stop exposing the otSdkDriver at https://github.com/mozilla/loop/blob/64be6385a1b0242d37c895c69350099397e1a5aa/add-on/panels/js/conversation.jsx#L241